sources/install.sh

```diff
@@ -3326,6 +3326,7 @@ function package_base() {
   install_exegol-history
   install_logrotate
   fapt openjdk-17-jre
+  install_chrome
```

Comment:
Here the wrong function name is used. It's more like `install_chromium`

Reply:
Done